dev/fattest.simplicity/src/componenttest/topology/impl/probe/CriuSupport.java

```diff
@@ -70,7 +79,7 @@ public static void main(String[] args) {
      * @param string
      */
     private static void err(String string) {
-        System.err.println("criuSupport> " + string);
+        System.err.println("string");
```

Comment:
You meant to use the var `string` here and not the literal value of "string" here.

Reply:
Good catch. Thanks.